### PR TITLE
Updated typo

### PR DESCRIPTION
**Pull request by jvishnuvardhan**
When we change input `test_images` from the currently selected image to other in the [tutorial](https://www.tensorflow.org/tutorials/keras/basic_classification), then the prediction will change and reports correctly. But, the static text at the end of the tutorial `And the model predicts a label of 2.` will not change.

So I changed the static text from `And the model predicts a label of 2.` to `And the model predicts a label as expected.` 
Thanks

### CODE REVIEW

**Comment by jvishnuvardhan** [comment]
@lamberta I have updated the file in master as you suggested. https://github.com/tensorflow/docs/pull/987

**Comment by lamberta** [comment]
Thank you! Closing this PR for that one
